Portal/Views/Shared/_Layout.cshtml

```diff
@@ -1,8 +1,10 @@
 ﻿@using Portal.Resources;
 @inject Microsoft.AspNetCore.Http.IHttpContextAccessor httpContextAccessor
+@inject IQueryStringHelper queryStringHelper
 
 @{
     var (oppositeCultureValue, oppositeCultureName) = CultureHelper.GetCultureInfo(httpContextAccessor);
+    var existingQueryStringParameters = queryStringHelper.RemoveCultureQueryString(Context.Request.QueryString.ToString());
```

Comment:
Since IQueryStringHelper is being injected, couldn't the HttpContext (Via HttpContextAccessor) be injected into that, and therefore no parameter needs to be passed into RemoveCultureQueryString

Reply:
The method informs us of what it's doing already.